docker/entrypoint.R

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env Rscript
+
+options <- commandArgs(trailingOnly = TRUE)
+outpack.orderly::orderly2outpack(options[[1]], options[[2]])
```

Comment:
this entrypoint does the migration once only, then exits? do we want a container that stays up and runs on some schedule?

Reply:
Probably worth using docopt here really, too

Reply:
also considered just relying on mountpoints (i.e. mount orderly to /orderly, outpack to /outpack and then use those?)

Reply:
this container is just used to bring up an instance of outpack metadata for testing the proxying functionality in OW. I guess if we bring docopt in and have a fuller interface we can have args for specifying one-off vs scheduled migrations